universities/organizations.json

```diff
@@ -361,33 +630,87 @@
   },
   {
     "id": "7264",
+    "icpc_id": "7264",
     "name": "University of Göttin",
     "formal_name": "University of Göttingen",
-    "country": "DEU"
+    "country": "DEU",
+    "url": "http://www.uni-goettingen.de",
+    "location": {
+      "longitude": 9.905548,
+      "latitude": 51.545483
+    }
+  },
+  {
+    "id": "7431",
+    "icpc_id": "7431",
+    "name": "AAU",
+    "formal_name": "Aalborg University",
+    "country": "DNK",
+    "url": "http://aau.dk",
+    "location": {
+      "longitude": 9.9850,
+      "latitude": 57.0150
+    }
   },
   {
     "id": "7464",
+    "icpc_id": "7464",
     "name": "Brunel",
     "formal_name": "Brunel University London",
-    "country": "GBR"
+    "country": "GBR",
+    "url": "http://www.brunel.ac.uk/",
+    "location": {
+      "longitude": -0.12574,
+      "latitude": 51.50853
+    }
   },
   {
     "id": "7477",
+    "icpc_id": "7477",
     "name": "Vilnius Gediminas TU",
     "formal_name": "Vilnius Gediminas Technical University",
-    "country": "LTU"
+    "country": "LTU",
+    "url": "http://www.vgtu.lt/en/",
+    "location": {
+      "longitude": 25.287514,
+      "latitude": 54.682789
+    }
+  },
+  {
+    "id": "7488",
+    "icpc_id": "7488",
+    "name": "NUI Maynooth",
+    "formal_name": "Maynooth University",
```

Comment:
Should it also be added to the Readme? 🙂

Reply:
Forgot about this. Will run the table generate script in a later commit again.